libs/metatron/pom.xml

```diff
@@ -54,6 +54,90 @@
         </exclusion>
       </exclusions>
     </dependency>
```

Comment:
This `libs` dir and contents can be removed. The gradle task that creates these poms was [recently removed](https://github.com/alphagov/verify-proxy-node/commit/d011c607376ec493329e312e520cb02fdbea9d7f) - suspect this was generated and commited before that happened.

Reply:
👍